staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -205,3 +205,130 @@ type DaemonSetList struct {
 	// A list of daemon sets.
 	Items []DaemonSet `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
+
```

Comment:
This is not generated. It should be in your initial commit.

Reply:
you are right. Same for `staging/src/k8s.io/api/apps/v1/register.go` too.